### DISCUSSION TITLE
Adding vehicles to initial point spawn

### DISCUSSION THREAD

**Discussion by Tetlys** [Vdauphin/HeartsAndMinds#1427] [Question and answer] [answered]
Hey! I was just wondering if it were possible to add vehicles into the initial point activation instead of them coming from the nearest occupied point. I could be mistaken but from looking all vehicle spawning happens outside of the objective and then move in via waypoint. 

I'm putting together a version on South Asia, with points up to 5-10km away, so havnig them drive in from the nearest points maybe won't work. Don't need em for all points, just some. I've noticed the spawn more code, but believe this is also calling vehicles from other points.

Is this as simple as adding a vehicle into the units array? or is there another way to have a few vehicles spawn per point on initial activation.

**Comment by Tetlys** [comment]
Is this also the same for patrols? That they would travel from the nearest point? I could change the patrol vehicles into aircraft to have them travel between points, shouldn't cause too much pain.  However I'd still like the Odd ground vehicle to be at each point

**Reply by Vdauphin** [reply]
> Is this also the same for patrols? That they would travel from the nearest point? 

Yes

>  I could change the patrol vehicles into aircraft to have them travel between points

Yes

**Comment by Vdauphin** [comment, marked as answer]
Hello,

> Is this as simple as adding a vehicle into the units array? 

No, this has been design like that to avoid player see vehicle spawn in front of them
I don't see easy way honestly, you need to change this file https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/city/activate.sqf

Cheers

**Comment by Tetlys** [comment]
Thanks! Having a play seems I can swap out the start city with the end city, ugly at close range but with our activation range should be okay.

Thanks for creating such a great game mode!

**Reply by Vdauphin** [reply]
Your welcome :) and good luck!
